archeo/theme.json

```diff
@@ -166,11 +166,6 @@
 	},
 	"styles": {
 		"blocks": {
-			"core/button": {
-				"typography": {
-					"textTransform": "uppercase"
-				}
```

Comment:
Why did you pull this out?  Was the UPPERCASE transform not in the original design?  It changes existing button styles.